examples/open_loop_control_urdfbot/generate.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# generate series of actions for fetch agent on different parts such as moving, arm and gripper.
+
+import numpy as np
+import pandas as pd
+
+wheel_joints = ["wheel_joint_r", "wheel_joint_l"]
+arm_joints = ["arm_joint_0", "arm_joint_1", "arm_joint_2", "arm_joint_3", "arm_joint_4", "arm_joint_5", "arm_joint_6"]
+gripper_joints = ["gripper_finger_joint_r", "gripper_finger_joint_l"]
+other_joints = ["head_pan_joint", "head_tilt_joint"]
+
+arm_poses = np.pi / 180 * np.array([
+    np.array([0, 0, 0, 0, 0., 0, 0.]),
+    np.array([-63.87276204, -3.9247609, 89.93145552, 78.6693971, -42.55529432, 80.11668849, 102.91353325]),
+    np.array([-67.08168819, 84.27042205, 22.91831181, 95.72957795, 0., 90., 0.]),
+    np.array([-81.94213203, 12.01206017, 107.03768346, 101.74355343, -15.63544527, 75.46713599, 115.29400528])
+])
+
+
+def add_action(action_map, joint, name):
+    action_map["joint." + joint] = np.array([name], dtype=np.float32)
+
+
+def add_actions(actions_file, move_forward=0.0, move_right=0.0, gripper_state=50.0, arm_current=0, arm_next=0,
```

Comment:
it would be more clear if this function was written in a more functional style:
```
def get_action(move_forward=0.0, move_right=0.0, gripper_displacement=50.0, arm_pose_blend_weights=np.array([0.0, 0.0, 0.0, 0.0])):
    action = {}

    action["wheel_joint_l"] = move_forward - move_right
    action["wheel_joint_r"] = move_forward + move_right

    arm_pose = # matrix multiply arm_poses and arm_pose_blend_weights

    action["arm_joint_0"] = arm_pose[0]
    action["arm_joint_1"] = arm_pose[1]

    # ...

    return action
```

Reply:
fixed